src/controllers/bookingForm.js

```diff
@@ -0,0 +1,43 @@
+const nodemailer = require('nodemailer');
+require("env2")("/home/nareman/Desktop/mdf/Move-Dance-Feel/.env");
+const password = process.env.pass;
+const fs = require('fs');
+const MUSTACHE = require('mustache');
+exports.get = (req, res) => {
+  res.render('booking', {
+    js: ['domBoking'],
+    css: 'booking',
+  });
+};
+exports.post = (async (req, res) => {
+  const {
+    name, email, experince, message, int,
+  } = req.body;
+  const data = {
+    name, email, experince, message, int,
+  };
+  async function main() {
+    const transporter = nodemailer.createTransport({
+      service: 'gmail',
+      auth: {
+        user: 'naremanmohhilles@gmail.com', // emily email
+        pass: password, // pass of emily
+      },
+    });
+
+    const template = fs.readFileSync(
+      '/home/nareman/Desktop/mdf/Move-Dance-Feel/public/html/book.html',
```

Comment:
Afraid this is specific to you, so if you substitute in  __dirname (that gives you path to the directory this file is in aka the controllers' directory) +  <a relative path from the controllers' directory to the book.html file>

Reply:
> Afraid this is specific to you, so if you substitute in __dirname (that gives you path to the directory this file is in aka the controllers' directory) + <a relative path from the controllers' directory to the book.html file> it'll be generic and we can all use it. :)

fixed it